upload_assets.py

```diff
@@ -31,29 +31,43 @@ def upload_files_to_bucket(self, bucket_name, geo_projects_directory):
                         log.warning('Could not determine species name. Skipping project %s.', project_uuid)
                     else:
                         key = f'{key_prefix}{project_uuid}.{species_name}{file_extension}'
-                        content_disposition = f'attachment;filename="{project_uuid}.{species_name}{file_extension}"'
                         obj = self.s3.Object(bucket_name, key)
-
-                        log.info(f'Uploading %s to s3://%s/%s.', matrix_path, bucket_name, key)
-                        with open(matrix_path, 'rb') as mf:
-                            obj.put(Body=mf,
-                                    ContentDisposition=content_disposition,
-                                    ContentType='application/zip, application/octet-stream')
-                        obj.Acl().put(AccessControlPolicy={
-                            'Owner': {
-                                'DisplayName': 'czi-aws-admins+humancellatlas',
-                                'ID': '76fe35006be54bbb55cf619bf94684704f14362141f2422a19c3af23e080a148'
-                            },
-                            'Grants': [
-                                {
-                                    'Grantee': {
-                                        'URI': 'http://acs.amazonaws.com/groups/global/AllUsers',
-                                        'Type': 'Group'
-                                    },
-                                    'Permission': 'READ'
-                                }
-                            ]
-                        })
+                        try:
+                            last_mod = obj.last_modified
+                        except self.s3.meta.client.exceptions.ClientError as e:
+                            if e.response['Error']['Code'] == '404':
+                                remote_last_mod = 0
+                            else:
+                                raise e
```

Comment:
```suggestion
                                raise
```

And study why that's preferred.